<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg</class>
 <widget class="QWidget" name="reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#reg {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #3498db, stop:1 #2c3e50);
    border-radius: 10px;
}

/* 标题样式 */
QLabel#label {
    color: white;
    font-weight: bold;
    font-size: 28px;
    padding: 10px;
    margin-bottom: 20px;
}

/* 输入框容器样式 */
QWidget#widget_4 {
    background-color: rgba(255, 255, 255, 0.1);
    border-radius: 10px;
    padding: 15px;
}

/* 标签样式 */
QLabel {
    color: white;
    font-size: 16px;
    min-width: 80px;
}

/* 输入框通用样式 */
QLineEdit {
    background-color: rgba(255, 255, 255, 0.9);
    border: 2px solid #ecf0f1;
    border-radius: 8px;
    padding: 8px 12px;
    font-size: 16px;
    color: #2c3e50;
    min-width: 200px;
    selection-background-color: #3498db;
    selection-color: white;
}

QLineEdit:focus {
    border: 2px solid #3498db;
    background-color: white;
}

QLineEdit::placeholder {
    color: #95a5a6;
    font-style: italic;
}



/* 注册按钮样式 */
QPushButton#reg_2 {
    background-color: #2ecc71;
    color: white;
    border: 2px solid #27ae60;
    border-radius: 8px;
    padding: 10px;
    font-size: 16px;
    font-weight: bold;
    margin-top: 15px;
}

QPushButton#reg_2:hover {
    background-color: #27ae60;
}

QPushButton#reg_2:pressed {
    background-color: #219653;
}

/* 布局间距调整 */
QGridLayout {
    margin: 15px;
}

QGridLayout &gt; QWidget {
    margin: 5px;
}
	QMessageBox QLabel {
   	 	color: black;  /* 设置消息文本为黑色 */
    	font-weight: normal;
	}
QMessageBox QPushButton {
    background-color: #f0f0f0;
    color: black;
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 5px 10px;
    min-width: 80px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>注册界面</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_user">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="placeholderText">
         <string>请输入用户名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_password2">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb2_2">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>注册密码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb1_2">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="reg_2">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_password1">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>25</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb2_3">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
